Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM node:20.18.0-bullseye-slim
 
+LABEL org.opencontainers.image.source https://github.com/VKCOM/devicehub
+LABEL org.opencontainers.image.title DeviceHub
+LABEL org.opencontainers.image.vendor VKCOM
+LABEL org.opencontainers.image.description "Control and manage Android devices from your browser."
```

Comment:
and ios, btw

Reply:
Then this must be fixed in the repo description since this is where I took this description from

Reply:
Fixed